pyleague/league.py

```diff
@@ -3,12 +3,23 @@
 import typing
 from typing import Optional
 
-from pyleague import simulation
-from pyleague.models import *
+import simulation
+from models import *
 
 
-class League:
+class League:    
+    """This is a class that conceptually emulates a real-world football league. It contains the attributes and methods that would be necessary in a real-world league. 
+
+    :param name: name of the league 
+    :type name: str
+    :param teams: list of Team objects
+    :type teams: list
+    ...
+    :raises ValueError: if the team objects are not all unique in terms of their .identifier attribute, then a ValueError will be raised  
```

Comment:
```suggestion
    :raises ValueError: if the team objects are not all unique in terms of their .identifier attribute
```
No need to repeat this :)